src/guides/wallet.md

```diff
@@ -0,0 +1,164 @@
+
+`hsd` implements [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)  as a method of generating unlimited accounts deterministically. This adds additional dimensions to the hierarchy described above, meaning the same seed that can recover all your keys can also recover all your addresses.
+
+Each account also comes with its own "extended public key," a piece of data that can be used to generate all public keys for that account in deterministic fashion. This means, for instance, that a business can create limitless deposit addresses for its users without having to touch its critical private keys or seed. Remember that public keys can be used for receiving HNS, but not for spending, so a public key falling into the wrong hands will not immediately result in theft.
+
+### Watch Only Wallets
+Speaking of not touching private keys, hsd gives you the option to create wallets that are "watch only." Watch only wallets don't contain any private keys, which means they can't be used to spend the HNS they receive. However, they work perfectly fine for creating addresses, receiving HNS, and detecting incoming transactions. Using watch only wallets where appropriate reduces the risk of your keys and HNS being stolen and is good security practice.
+
+Accounts always inherit the watch only behavior of their parent wallet. In other words, a watch only wallet will have exclusively watch only accounts while a regular wallet will have only regular accounts. Accordingly, you can't import private keys into a watch only wallet or public keys into regular wallets. If you try to mix and match watch only wallets and keys with hsd, you're gonna have a bad time.
+
+
+### API Authentication
+`hsd` runs as a server and allow you to interact with your wallets via a REST API. It also allows you protect wallets from unauthenticated requests by running the server with the wallet-auth option. Each wallet you create has a token value that must be passed in with each request. Tokens, like accounts and keys, can also be deterministically generated using your HD seed. This means you can change the token on a wallet as often as you'd like.
+
+### Recovery
+By using the HD standards mentioned above, hsd allows one to easily restore or transfer their entire wallet to different wallet implementations. By providing just the mnemonic, one can fully recover their wallet to a fresh instance of `hsd` or any other software that properly implements BIP33, BIP39, and BIP44, like the Trezor hardware wallet.
+
+*Above borrowed from [Wallets and Accounts and Keys, Oh My!](https://bcoin.io/guides/wallets.html) by Daniel McNally*
```

Comment:
I would mention this at the very top of this guide, and use stronger language, e.g. "this guide is heavily influenced by such and such....much of it is copied verbatim with name changes from Bitcoin to Handshake." Or something like this...